pkg/base/linuxutils/lsblk/lsblk.go

```diff
@@ -178,7 +173,7 @@ func (l *LSBLK) SearchDrivePath(drive *api.Drive) (string, error) {
 	vid := drive.VID
 	pid := drive.PID
 	for _, l := range lsblkOut {
-		if strings.EqualFold(l.Serial, sn) && strings.EqualFold(l.Vendor, vid) &&
+		if strings.EqualFold(l.Serial, sn) && strings.Contains(l.Vendor, vid) &&
 			strings.EqualFold(l.Model, pid) {
```

Comment:
Need code change here to handle the PID.
In our test, for the same disk, the VID and PID format is different from lsblk and drive CR. 

 l.Vendor  is  "ATA     " while  vid (drive.VID) is “ATA”
l.Model  is "HGST HUS728T8TALE6L0" while pid (drive.PID) is “HGST HUS728T8TAL”

Reply:
Agree!
Updated the changes, please take a look